README.org

```diff
@@ -32,6 +32,38 @@ the environment variable =ZMQ_VERSION=, e.g. to something like
 ZMQ_VERSION=4.3.0
 #+END_SRC
 
+*** Windows
+
+It is possible to use the included build chain on Windows using the [[https://www.msys2.org/][MSYS2]] MinGW
+tools.
+
+Install the 64-bit toolchain inside the MSYS2 shell via
+#+BEGIN_SRC shell
+pacman -S base-devel
+pacman -S git
+pacman -S mingw-w64-x86_64-gcc
+#+END_SRC
+
+Note: If you are using the official Git for Windows instead of MSYS2 Git, make
+sure to set
+
+#+BEGIN_SRC shell
+git config --global core.autocrlf false
+#+END_SRC
+
+during the build to avoid EOL issues and set it back to =true= (the default on
+Windows) when you are done building.
+
+Start the build from an MSYS2 MinGW 64-bit shell via =make=.
+
+The =libzmq.dll= is built to =libzmq/build/v*/bin= and needs to be copied to the
+top level directory next to the =emacs-zmq.dll= to be found by the Windows DLL
+lookup machinery.
+
+Note that you need to have Emacs compiled =--with-modules= to be able to load
+shared modules. See instructions [[https://sourceforge.net/p/emacsbinw64/wiki/Build%2520guideline%2520for%2520MSYS2-MinGW-w64%2520system/][here]] for a way to do this using the mentioned
```

Comment:
I'm getting a page not found error on the link. It should be `%20` instead of `%2520`.

Also I took a look at the github project of the website and it looks like he has not released any binaries for Emacs 26.1 and based on this [issue](https://github.com/zklhp/emacs-w64/issues/7), he doesn't intend to. Supposedly because the official Emacs Windows binaries are good enough. And in his [releases](https://github.com/zklhp/emacs-w64/releases) he is building `--with-modules` so I'm wondering if the official Emacs 26.1 (https://ftp.gnu.org/gnu/emacs/windows/emacs-26/) or the pretest Emacs 26 (https://alpha.gnu.org/gnu/emacs/pretest/windows/emacs-26/) have module support? If they do, we can just point users there.

Reply:
Ah, good catch, will adapt. The reason I pointed to that page is that it is where I took the instructions to build Emacs using MinGW - it was not about using that built release. AFAICS the Emacs I used when first trying this was the official one and it did not have module support - see https://github.com/dzop/emacs-zmq/issues/3#issuecomment-461852855. There might be an official one but I did not find any docs about this or one that worked.